### DISCUSSION TITLE
neovim visionary awards 2023

### DISCUSSION THREAD

**Discussion by 1412031** [folke/edgy.nvim#7] [General]
Just dropped by to say a hi and that you're absolutely killing it.. huge respects for how impactful your work has turned to be and the sheer volume of all of it altogether is impressive, and factoring in how inspired and innovative the actual mechanics are coupled with how determined you turned out to be is pretty admirable.

Props man respect you're really killing it, a major part of my setup at the time being

**Comment by folke** [comment]
Thank you!!
